checkov/common/sca/commons.py

```diff
@@ -47,3 +48,24 @@ def normalize_twistcli_language(language: str) -> str:
 
 def should_run_scan(runner_filter_checks: Optional[List[str]]) -> bool:
     return not (runner_filter_checks and all(not (check.startswith("CKV_CVE") or check.startswith("BC_CVE") or check.startswith("BC_LIC")) for check in runner_filter_checks))
+
+
+def get_lowest_fix_version(vulnerability_details: dict[str, Any]) -> str:
+    if "cveStatus" in vulnerability_details:
+        return str(vulnerability_details["cveStatus"])
+
+    lowest_fixed_version = UNFIXABLE_VERSION
+    package_version = vulnerability_details["package_version"]
+    fixed_versions: list[packaging_version.Version | packaging_version.LegacyVersion] = []
+    status = vulnerability_details.get("status") or "open"
```

Comment:
can you create a global const for status `open` next to `UNFIXABLE_VERSION` at the top